File: Python/Algorithm/programmers/adder.py
# [adder.py]
# 두 정수 사이의 합

# First code
def solution1(a, b):
    if a <= b:
        answer = sum(list(range(a, b+1)))
    else:
        answer = sum(list(range(b, a+1)))
    return answer

# Improved code
def solution2(a, b):
    answer = sum(list(range(min(a,b), max(a,b)+1)))
    return answer

File: Java/Algorithm/programmers/adder.java
// [adder.java]
// 두 정수 사이의 합

class Solution {
  public long solution(int a, int b) {
      long answer = 0;
      // 1. 작은수부터 큰수까지의 범위를 더한다.
      for (int i = Math.min(a,b); i <= Math.max(a,b); i++){
          answer += i;
      }
      return answer;
          
  }
}